format/stmt.go

```diff
@@ -60,13 +89,13 @@ func (p *printer) stmt(s stmt.Stmt) {
 			}
 			p.stmt(c.Body)
 		}
-		p.buf.WriteString("}")
+		p.buf.WriteString("}\n")
```

Comment:
Same question as above about final newlines.

Reply:
done

Reply:
(it was a misguided attempt at preparing for `ngfmt`)